resources/views/fields/checkbox.blade.php

```diff
@@ -1,14 +1,24 @@
-<div class="{{ $field->divClass ?? config('livewire-forms.defaults.divClass') }}">
-    <input
-        @include('livewire-forms::fields.binding')
-        id="{{ $field->getName() }}"
-        type="checkbox"
-        class="{{ $field->class ?? config('livewire-forms.defaults.inputClass') }}"
-        name="{{ $field->getName() }}"
-        @if ($field->dusk) dusk={{ $field->dusk }} @endif
-    >
+<div @class([
+    $field->divClass ?? config('livewire-forms.defaults.divClass'),
+    $field->colClass ?? config('livewire-forms.defaults.colClass')])
+>
+    <div class="form-check">
```

Comment:
Evt. de .form-check class ook nog in de config steken en via een property overschrijfbaar maken?